content/concept-stand-alone-mode.md

```diff
@@ -0,0 +1,103 @@
+Stand-Alone Mode
+===============================================================================
+
+You can run `rippled` in stand-alone mode without a consensus of trusted servers. In stand-alone mode, `rippled` does not communicate with any other servers in the Ripple peer-to-peer network, but you can do most of the same actions on your local server only. Stand-alone provides a method for testing `rippled` behavior without being tied to the live network. For example, you can [test the effects of Amendments](concept-amendments.html#testing-amendments) before those Amendments have gone into effect across the decentralized network.
+
+When you run `rippled` in stand-alone mode, you have to tell it what ledger version to start from:
+
+* Create a [new genesis ledger](#new-genesis-ledger) from scratch.
+* [Load an existing ledger version](#load-saved-ledger) from disk.
+
+**Caution:** In stand-alone mode, you must [manually advance the ledger](#advancing-ledgers-in-stand-alone-mode).
+
+New Genesis Ledger
+-------------------------------------------------------------------------------
+In stand-alone mode, you can have `rippled` create a new genesis ledger. This provides a known state, with none of the ledger history from the production Ripple Consensus Ledger. (This is very useful for unit tests, among other things.)
+
+* To start `rippled` in stand-alone mode with a new genesis ledger, use the `-a` and `--start` options:
```

Comment:
This tutorial is not the right place to go into an explicit definition of the flags. I've linked to the rippled wiki page for now. Anything not there will have to wait for https://ripplelabs.atlassian.net/browse/DOC-368